config/application.rb

```diff
@@ -2,17 +2,20 @@
 
 require_relative 'boot'
 
+# require 'rails'
```

Comment:
Mike - do you think we need to deploy and test infrastructure-integration-tests ?

Reply:
I deployed to stage and then ran an integration test to create an object and it got a druid, so we should be good to go.